src/core/modal.slice.ts

```diff
@@ -83,6 +81,12 @@ const modalSlice = createSlice({
         removeModalIdFromUrl(state);
         returnFocusElement();
       }
+      // Enables background scrolling to use when last modal is closed
+      // console.log(getConf("modalIds", configuration));
```

Comment:
Do we want to have uncommented console.log's in the code?

Reply:
Well spotted! I am removing it :)))